src/App.js

```diff
@@ -33,7 +34,9 @@ const App = () => {
     <PaperProvider theme={theme}>
       <SafeAreaView style={styles.safeAreaView}>
         <React.Fragment>
-          <AddWebApp />
+          <GDStore.Container>
+            <AddWebApp />
```

Comment:
no we dont want gdstore here, it will add many dependencies which is bad for dyanmic loading.
actually there's an issue to move AddWebApp deeper in dom tree to Router.js

Reply:
done